.github/workflows/CI.yml

```diff
@@ -41,15 +73,21 @@ jobs:
           run: npm run build
 
         - name: E2E tests
-          run: npm run test:e2e
+          run: npx playwright test --project webkit
           env:
-            E2ETEST_POSTGRES_URL: ${{ secrets.E2ETEST_POSTGRES_URL }}
-            E2ETEST: 1
+            E2ETEST2_POSTGRES_URL: ${{ secrets.E2ETEST2_POSTGRES_URL }}
+            E2ETEST: 2
```

Comment:
Why is this `2` value given? Can't we just give `1` since unmodified `database.js` would use value of `E2ETEST_POSTGRES_URL` env var which would be set to `secrets.E2ETEST2_POSTGRES_URL` and thus use different value from the other job.